internal/scan/scan.go

```diff
@@ -282,7 +282,7 @@ func (s *Scanner) getCapabilities() (*chartutil.Capabilities, error) {
 		cmd.Stdout = &out
 		err = cmd.Run()
 		if err != nil {
-			log.Println("error getting tiller version")
+			log.Println("Info: Tiller capability not detected (cvscan)")
```

Comment:
Does the standard error that comes from these commands provide useful information? If so, capturing the std error buffer from `cmd.Stderr` could be useful and include that in the log output.

If the information is not useful, then what you've done here makes sense.

Reply:
Good question, It dumped: `Error: could not find tiller` - seemed just as misleading as what we had.

Reply:
How about this:
```suggestion
			log.Println("Info: Tiller is unavailable in this cluster, so the Tiller version can not be determined (cvscan)")
```

Reply:
I'll be "that guy" - "cannot" preferred over "can not" because "not" isn't a part of another construction: https://grammarist.com/usage/cannot-or-can-not/